packages/framework-provider-local-infrastructure/test/scheduler.test.ts

```diff
@@ -0,0 +1,70 @@
+import { BoosterConfig } from '@boostercloud/framework-types'
+import { describe } from 'mocha'
+import { fake, replace, restore } from 'sinon'
+import { configureScheduler } from '../src/scheduler'
+import { expect } from '../test/expect'
+import * as scheduler from 'node-schedule'
+
+describe('scheduler', () => {
+  let mockConfig: any
+
+  const userProject = {
+    boosterTriggerScheduledCommand: () => {
+      console.log('triggering scheduled command...')
+    },
+  }
+
+  describe('configureScheduler', () => {
+    beforeEach(() => {
+      replace(scheduler, 'scheduleJob', fake())
+    })
+
+    afterEach(() => {
+      restore()
+    })
+
+    it('should schedule all commands with the corresponding crons', async () => {
```

Comment:
@NickSeagull, regarding more complex cases, I can think of this one for this test file:

```
it('should only schedule commands where `scheduledOn` is defined'

(...)
```

This would test the filtering feature of this line specifically:
https://github.com/boostercloud/booster/pull/1058/files#diff-0871e6dff27016ddf8f5c7753a016ebcf1717fa0bfcd7bad37bd6c06de4029b3R13


I can also think of testing multiple cron expressions of higher complexity, such as the examples seen here:
https://www.baeldung.com/cron-expressions
https://docs.oracle.com/cd/E12058_01/doc/doc.1014/e12030/cron_expressions.htm

Did you have other kinds of tests in mind for this file?

Reply:
Awesome, that first suggestion sounds great! 🙏 

If you have the time and energy, maybe trying [fast-check](https://github.com/dubzzz/fast-check) to generate some cron expressions could be a good idea. No need to go overboard on this one, as fast-check is a pretty deep library :)